sdk/eventhub/Azure.Messaging.EventHubs.Processor/src/Diagnostics/EventProcessorClientEventSource.cs

```diff
@@ -39,264 +39,132 @@ internal EventProcessorClientEventSource() : base(EventSourceName, EventSourceSe
         /// </summary>
         ///
+        /// <param name="partitionId">The identifier of the Event Hub partition whose processing is taking place.</param>
         /// <param name="identifier">A unique name used to identify the event processor.</param>
+        /// <param name="eventHubName">The name of the Event Hub that the processor is associated with.</param>
+        /// <param name="consumerGroup">The name of the consumer group that the processor is associated with.</param>
         ///
-        [Event(1, Level = EventLevel.Informational, Message = "Starting a new event processor instance with identifier '{0}'.")]
-        public virtual void EventProcessorStart(string identifier)
-        {
-            if (IsEnabled())
-            {
-                WriteEvent(1, identifier ?? string.Empty);
-            }
-        }
-
-        /// <summary>
-        ///   Indicates the total number of ownership records returned by storage manager.
-        /// </summary>
-        ///
-        /// <param name="count">Total number of ownership records returned by storage manager.</param>
-        ///
-        [Event(2, Level = EventLevel.Informational, Message = "Storage manager returned '{0}' ownership records.")]
-        public virtual void CompleteOwnershipRecordsCount(int count)
-        {
-            if (IsEnabled())
-            {
-                WriteEvent(2, count);
-            }
-        }
-
-        /// <summary>
-        ///   Indicates that an exception was encountered while retrieving ownership records.
-        /// </summary>
-        ///
-        /// <param name="fullyQualifiedNamespace">The fully qualified Event Hubs namespace that the processor is associated with.</param>
-        /// <param name="eventHubName">The name of the Event Hub that the processor is connected to, specific to the Event Hubs namespace that contains it.</param>
-        /// <param name="consumerGroup">The name of the consumer group this event processor is associated with.  Events will be read only in the context of this group.</param>
-        /// <param name="errorMessage">The message for the exception that occurred.</param>
-        ///
-        [Event(3, Level = EventLevel.Error, Message = "An exception occurred while retrieving ownership records. (FullyQualifiedNamespace: '{0}'; EventHubName: '{1}'; ConsumerGroup: '{2}'; ErrorMessage: '{3}')")]
-        public virtual void ListOwnershipAsyncError(string fullyQualifiedNamespace,
-                                                    string eventHubName,
-                                                    string consumerGroup,
-                                                    string errorMessage)
+        [Event(20, Level = EventLevel.Verbose, Message = "Starting to process a batch of events for partition '{0}' by processor instance with identifier '{1}' for Event Hub: {2} and Consumer Group: {3}.")]
```

Comment:
Why are we starting from 20?

Reply:
Because the previous version ended in the late teens.   Since this new version bears the same diagnostic source as the old, we don't want to stomp or mangle data that was captured with the GA version of the library.